docs/source/release_notes/release-notes.rst

```diff
@@ -23,6 +23,25 @@ Changelog
 .. <<<<------------------------------------------------------------------------------------------------------------->>>>
 
 =======
+v0.7.0
+=======
+
+.. list-table::
+    :widths: 5 40
+    :header-rows: 1
+
+    *   - Type
+        - Change
+
+    *   - |changed|
```

Comment:
If we never find a use for changed then I'm happy with that - but a big red breaking is necessary in my view.